gtsam/geometry/Pose2.h

```diff
@@ -192,15 +192,15 @@ class Pose2: public LieGroup<Pose2, 3> {
   }
 
   /// Derivative of Expmap
-  GTSAM_EXPORT static Matrix3 ExpmapDerivative(const Vector3& v);
+  static Matrix3 ExpmapDerivative(const Vector3& v);
 
   /// Derivative of Logmap
-  GTSAM_EXPORT static Matrix3 LogmapDerivative(const Pose2& v);
+  static Matrix3 LogmapDerivative(const Pose2& v);
 
   // Chart at origin, depends on compile-time flag SLOW_BUT_CORRECT_EXPMAP
-  struct ChartAtOrigin {
-	GTSAM_EXPORT static Pose2 Retract(const Vector3& v, ChartJacobian H = {});
-	GTSAM_EXPORT static Vector3 Local(const Pose2& r, ChartJacobian H = {});
+  struct GTSAM_EXPORT ChartAtOrigin {
```

Comment:
I think this is incorrect since `ChartAtOrigin` is an inner class.

Reply:
ChartAtOrigin is inner class. As I discovered from the errors. It also require an export. Other wise it missing their functions symbols. The GTSAM_EXPORT on Pose2 class it not enough.